aviary/subsystems/propulsion/propeller/propeller_performance.py

```diff
@@ -289,7 +416,10 @@ def setup(self):
 
 class PropellerPerformance(om.Group):
     """
-    Computation of propeller thrust coefficient based on Hamilton Standard.
+    Computation of propeller thrust coefficient based on the Hamilton Standard model or a user
+    provided propeller map. Note that a propeller map allows either the helical Mach number or
+    free stream Mach number as input. This infomation will be detected automatically when the 
+    propeller map is loaded into memory.
```

Comment:
I think it would be useful to mention the #mach_type comment that you have to put in the file to detect this. Because right now it's not actually automatic, it relies on that flag in the comments.

Reply:
Now, the mach type is detected from header instead of comment.

For `general_aviation.prop`, the header is:

```
    J, Helical_Mach,         CP,         CT
```
For `PropFan.prop`, the header is:
```
  J,       Mach,         CP,         CT
```